awscli/customizations/opsworks.py

```diff
@@ -278,7 +278,8 @@ def validate_arguments(self, args):
 
         if args.infrastructure_class == 'ec2' and args.local:
             # make sure the regions match
-            region = json.loads(urlopen(IDENTITY_URL).read())['region']
+            response = urlopen(IDENTITY_URL).read()
```

Comment:
We should be able to simplify this and *always* decode from utf-8.  Even in the case of python2 we'd want that.